lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -1074,9 +1089,9 @@ public function getTimeFormatString()
      */
     public function fixSchemaElementName($schemaElementName)
     {
-        if (strlen($schemaElementName) > 30) {
+        if (strlen($schemaElementName) > $this->getMaxIdentifierLength()) {
```

Comment:
This method doesn't seem to be used anywhere. Why is it needed to modify it?

Reply:
This is irrelevant to the fix and is not used anywhere. I'm going to deprecate this method and remove it later. Please revert.

Reply:
The changes in the mentioned method have been reverted.